tests/test_monkeypatch.py

```diff
@@ -156,3 +156,69 @@ def test_extended_moderation_admin_no_redirect_invalid_mr_to_copy(self):
         self.assertEqual(ContentExpiry.objects.count(), 1)
         # We should still be redirected
         self.assertEqual(response.status_code, 302)
+
+    def test_extended_moderation_admin_update_existing_compliance_number_record(self):
+        """
+        If the target of the copy already has a compliance number, the record should be updated, rather than recreated
+        """
+        # Create db data for additional content expiry!
+        self.content_expiry_secondary = factories.PollContentExpiryFactory(
+            expires=self.expires_secondary,
+            version__state=PUBLISHED
+        )
+        self.moderation_request2 = ModerationRequestFactory(
+            collection=self.collection,
+            version=self.content_expiry_secondary.version,
+        )
+        self.root2 = RootModerationRequestTreeNodeFactory(
+            moderation_request=self.moderation_request2
+        )
+        ChildModerationRequestTreeNodeFactory(
+            moderation_request=self.moderation_request1, parent=self.root1
+        )
+
+        # Check that compliance numbers are different before we hit the copy endpoint!
+        self.assertNotEqual(ContentExpiry.objects.first().compliance_number,
+                            ContentExpiry.objects.last().compliance_number)
+
+        response = self.client.post(self.url + "&copy=compliance")
+
+        # Ensure request is a redirect as expected!
```

Comment:
This comment is outdated, 200 isn't a redirect!

Reply:
A bit confused about this one as the test expect a 302

https://github.com/EligaServices/djangocms-content-expiry/blob/64d777dd4584c5c88189d4ac8ce658df85330ef1/tests/test_monkeypatch.py#L187

Reply:
Hmm, not sure why I thought it was a 200 then, will resolve.